code/modules/antagonists/abductor/equipment/abduction_gear.dm

```diff
@@ -321,7 +326,7 @@
 	to_chat(user, span_notice("You switch the device to [mode==MIND_DEVICE_MESSAGE? "TRANSMISSION": "COMMAND"] MODE"))
 
 /obj/item/abductor/mind_device/afterattack(atom/target, mob/living/user, flag, params)
-	. = ..()
+	. = ..() | AFTERATTACK_PROCESSED_ITEM
```

Comment:
is the idea to add the flag when we check to see if we could act on someone?

Reply:
if its reasonable to suspect that a competent player would click on an item with this, then the flag is added

Reply:
alright